lib/pay/stripe/billable.rb

```diff
@@ -6,14 +6,24 @@ module Billable
       # Returns Stripe::Customer
       def stripe_customer
         if processor_id?
-          ::Stripe::Customer.retrieve(processor_id)
+          retrieve_customer_with_marketplace
         else
           create_stripe_customer
         end
       rescue ::Stripe::StripeError => e
         raise Error, e.message
       end
 
+      def retrieve_customer_with_marketplace
+        if marketplace_id?
+          ::Stripe::Customer.retrieve(processor_id, {
+            stripe_account: marketplace_id
+          })
+        else
+          ::Stripe::Customer.retrieve(processor_id)
+        end
```

Comment:
```suggestion
          ::Stripe::Customer.retrieve(processor_id, {
            stripe_account: marketplace_id
          })
```

I think that if `marketplace_id` is `nil`, this will work exactly the same, so the `if/else` is unnecessary.

Reply:
If that works as normal when passing `nil`, that would be fantastic and would simplify things a lot.

Reply:
https://stripe.com/docs/api/customers/retrieve

I can't see it anywhere in the docs, but it appears that `stripe_account` is an optional param on retrieve, so passing `nil` would work exactly the same.

I have also manually tested this on a demo connect app I have and passing `nil` retrieves a customer from the main stripe account, passing an `account_id` retrieves the customer from the connected account.

Reply:
Great, sounds good then. I know sometimes API gems like this will validate an argument like that, but as usual, the Stripe gem is really, really well built.